src/api/config.ts

```diff
@@ -14,7 +14,7 @@ export const defaultConfig: Config = {
     showLanding: true,
     sentryEnabled: false,
     captchaLogin: false,
-    nicknamesEnabled: false,
+    usernamesEnabled: false,
```

Comment:
could you change please naming to usernameEnabled
isUsernameEnabled

sounds wrong in form 'isUsernamesEnabled'